File: backend/MoneyManagerBackend/MoneyManagerBackend/Domains/Transaction.cs
using System;

namespace MoneyManagerBackend.Domains
{
    public class Transaction
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public double Amount { get; set; }
        public string Account { get; set; }
        
    }
}

File: src/components/Transaction.js

const Transaction = ({transaction}) => {
    return (
        <div >
            <p>Date: {transaction.date}</p>
            <p>Amount: {transaction.amount}</p>
            <p>Description: {transaction.description}</p>
            <p>Account: {transaction.account}</p>
        </div>
    )
}

export default Transaction
